composer.json

```diff
@@ -16,7 +16,9 @@
         }
     ],
     "require": {
-        "scheb/two-factor-bundle": "^3.2.0",
+        "php": "^7.1.3",
```

Comment:
`>= 7.1.3` would be better, I guess.

Reply:
Can you explain this? Both `symfony/symfony` and `scheb/two-factor-bundle` require `^7.1.3`.

Reply:
I was mistaken, it's fine this way.